src/components/Menus/__tests__/DropdownMenu.spec.js

```diff
@@ -34,6 +34,43 @@ it('renders DropdownMenu with icons and trigger correctly', () => {
   expect(tree).toMatchSnapshot()
 })
 
+it('renders DropdownMenu in open state correctly', () => {
+  const menuContainerSelector = "div[data-testid='menu-container']"
```

Comment:
There's a `getByTestId` helper you could use here(`import { getByTestId } from 'testing/utils'`). Though noticed you have the element type as part of the selector, let me know if `getByTestId` was missing something you needed and we can add it if so.